dkpro-core-io-lcc-asl/src/main/java/org/dkpro/core/io/lcc/LccReader.java

```diff
@@ -60,6 +61,13 @@
     @ConfigurationParameter(name = PARAM_SOURCE_ENCODING, mandatory = true, defaultValue = ComponentParameters.DEFAULT_ENCODING)
     private String sourceEncoding;
     
+    /**
+     * Whether sentences should be written by the reader or not.
+     */
+    public static final String PARAM_WRITE_SENTENCES = "writeSentences";
```

Comment:
Here you should take the value of ComponentParameters.PARAM_WRITE_SENTENCE instead of using a literal.